src/components/DataTable/Checkbox.js

```diff
@@ -0,0 +1,20 @@
+import  React from "react";
+import { Input } from "reactstrap";
+
```

Comment:
@petrKavulok Cant be the logic of handling the checkbox selection directly here? Since you dont need it anywhere in the top level component, since you just want to check/uncheck all the checkbox inputs. Then you can just pass a prop, which will tell the datatable, if this feature should be enabled or not

Reply:
hmm.. I don't know.. this is how it's used [here](https://github.com/TeskaLabs/seacat-admin-webui/pull/47/files#diff-5411226715b02cd6ec40122336055311801e9d7c14f4f9c7c000ea2c9e8a702eR291) @Pe5h4